subsys/bootloader/image/CMakeLists.txt

```diff
@@ -51,6 +51,12 @@ set(PROVISION_HEX          ${PROJECT_BINARY_DIR}/${PROVISION_HEX_NAME})
 include(${CMAKE_CURRENT_LIST_DIR}/../cmake/debug_keys.cmake)
 include(${CMAKE_CURRENT_LIST_DIR}/../cmake/sign.cmake)
 
+if (${CONFIG_SB_DEBUG_SIGNATURE_PUBLIC_KEY_LAST})
+  set(ALL_PUBLIC_KEY_FILES "${PUBLIC_KEY_FILES},${SIGNATURE_PUBLIC_KEY_FILE}")
```

Comment:
Please print a big fat warning here. It is really bad if this is enabled by mistake.

Reply:
Done